custom-vu/src/main/kotlin/jces1209/vu/action/ViewBoard.kt

```diff
@@ -104,4 +107,36 @@ class ViewBoard(
             }
         }
     }
+
+    private fun changeIssueStatus(board: BoardPage) {
+        if (random.random.nextFloat() < changeIssueStatusProbability) {
+            if (board is KanbanBoardPage) {

```

Comment:
Detecting the implementation of an abstract type breaks encapsulation. `BoardPage` is polymorphic and should hide the implementation details from its callers.
Why do we check this here? Is it because other implementations are not complete?

Reply:
Because only this board can move issues between columns

Reply:
1. Then let's add `Boolean canMoveIssues()` to `BoardPage`.
2. Are you sure? I move issues between columns on my Scrum board every day.